Week4Task1/Main.java

```diff
@@ -0,0 +1,470 @@
+/***
+ * This Java program allows users to perform number conversions between Binary, Octal, Decimal, and Hexadecimal formats. It validates input, converts values, stores the results, and allows users to view stored numbers or perform arithmetic operations on them. The program includes methods for input validation, string-to-decimal conversions, decimal-to-other-bases conversions, and user interactions with formatted output and color-coded messages. The application is designed for flexibility with options to select and manipulate different number formats.
+ * Owner: Mayank Aitan
+ * Date of Creation: 24/09/2024
+ * Date of Updation: 26/09/2024
+ */
+package Week4Task1;
+import java.util.Scanner;
+public class Main {
+
+    static String[] binaryValues = new String[100];
+    static String[] octalValues = new String[100];
+    static String[] decimalValues = new String[100];
+    static String[] hexValues = new String[100];
+    final static String RESET = "\u001B[0m";
+    final static String RED = "\u001B[31m";
+    final static String GREEN = "\u001B[32m";
+    final static String YELLOW = "\u001B[33m";
+    final static String BLUE = "\u001B[34m";
+    final static String CYAN = "\u001B[36m";
+    final static String BOLD = "\u001B[1m";
+    final static String UNDERLINE = "\u001B[4m";
+    static int binaryCount = 0, octalCount = 0, decimalCount = 0, hexCount = 0;
+    static Constant message = new Constant();
+    public static boolean sign = true;
+
+//Method for handling main oprations
+    @SuppressWarnings("static-access")
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
```

Comment:
using true directly is not good approach! -> make a true value and then put true